src/test/java/cloud/eppo/RuleEvaluatorTest.java

```diff
@@ -0,0 +1,285 @@
+package cloud.eppo;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import cloud.eppo.ufc.dto.EppoValue;
+import cloud.eppo.ufc.dto.OperatorType;
+import cloud.eppo.ufc.dto.SubjectAttributes;
+import cloud.eppo.ufc.dto.TargetingCondition;
+import cloud.eppo.ufc.dto.TargetingRule;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.Test;
+
+public class RuleEvaluatorTest {
```

Comment:
Mostly not new code; Largely ported from Android implementation ([source](https://github.com/Eppo-exp/android-sdk/blob/main/eppo/src/test/java/cloud/eppo/android/RuleEvaluatorTest.java))